simplelink/source/ti/devices/cc13x2_cc26x2/CMakeLists.txt

```diff
@@ -21,13 +21,23 @@ zephyr_library_sources(
 
 # Required for IEEE 802.15.4 support
 zephyr_library_sources_ifdef(CONFIG_IEEE802154_CC13XX_CC26XX
-	rf_patches/rf_patch_cpe_ieee_802_15_4.c)
+  rf_patches/rf_patch_cpe_multi_protocol.c)
+# Required for IEEE 802.15.4 support
+# (until driver is changed to use multi-protocol rf patch)
+zephyr_library_sources_ifdef(CONFIG_IEEE802154_CC13XX_CC26XX
+  rf_patches/rf_patch_cpe_ieee_802_15_4.c)
 # Required for RFCDoorbellSendTo which is not in ROM
 zephyr_library_sources_ifdef(CONFIG_IEEE802154_CC13XX_CC26XX driverlib/rfc.c)
 
+# Required for IEEE 802.15.4g support
+zephyr_library_sources_ifdef(CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ
```

Comment:
Is it really necessary to introduce `CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ` right now? It seems you can leave out lines 32-36 and introduce them when you tackle https://github.com/zephyrproject-rtos/zephyr/issues/26315. This way we don't need https://github.com/zephyrproject-rtos/zephyr/pull/28534, which can be confusing if the plan is just for the Kconfig parameter to be temporary before you move on to changing it to something else later.

Reply:
Working on it now.

Reply:
I would say so, because it's needed for zephyrproject-rtos/zephyr#29598.

I guess technically, I should re-open zephyrproject-rtos/zephyr#28534 before this is done though, so we can do the usual regression test dance.

Reply:
Given you are now making progress on zephyrproject-rtos/zephyr#29598, I am ok with introducing this. I'll move out of the way.